dev/dags/dbt/model_version/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
We currently have the license. here:
https://github.com/astronomer/astronomer-cosmos/blob/main/LICENSE

So we don't need this one

Reply:
Ok.